meson.build

```diff
@@ -213,6 +213,13 @@ foreach tuple : _customdirs
   customdir = get_variable(dir_option)
   if customdir == ''
     customdir = def_path
+  else
+    if not customdir.startswith('/')
+      error('User provided @0@, \'@1@\' is not an absolute path.'
+          .format(dir_option, customdir))
+    endif
+    # Strip all leading/trailing and re-add only the leading one.
+    customdir = / customdir.strip('/')
```

Comment:
humn... I don't think this works as there are no left side operand.... you'd need to do '' / customdir.strip('/')

Reply:
indeed:

```
kmod $ meson setup -Dmoduledir=/test/foo build6
...
meson.build:222:16: ERROR: Unknown statement.
```

Reply:
Fixed this one

Reply:
Thanks :heart: 

Would it make sense to add a few `meson setup` permutations to our CI (without building)?

Reply:
yeah, not a bad idea

Reply:
Opened https://github.com/kmod-project/kmod/pull/245